policy/modules/services/zfs.te

```diff
@@ -0,0 +1,120 @@
+policy_module(zfs)
+
+########################################
+#
+# Declarations
+#
+
+attribute_role zfs_roles;
+
+type zed_t;
+type zed_exec_t;
+init_daemon_domain(zed_t, zed_exec_t)
+role zfs_roles types zed_t;
+
+type zfs_t;
+type zfs_exec_t;
+application_domain(zfs_t, zfs_exec_t)
+init_daemon_domain(zfs_t, zfs_exec_t)
```

Comment:
I'm very unfamiliar with zfs; how is this used as a daemon?

Reply:
Ah, this should just be `init_system_domain()` instead. The `application_domain()` can be removed as well since that is also called by the former.